charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -142,20 +149,55 @@ spec:
         volumeMounts:
           - mountPath: /config
             name: server-config
+      - name: jvm-client-options
```

Comment:
The test is failing now because it is asserting that there are two initContainers; this one makes three.

Reply:
I think tests were already failing before that 😄  I haven't updated them yet as this started out being somewhat what exploratory. I probably won't be able to get to it today.